packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -162,7 +165,7 @@ const callComponentWillUnmountWithTimer = function(current, instance) {
 };
 
 // Capture errors so they don't interrupt unmounting.
-function safelyCallComponentWillUnmount(current, instance) {
+export function safelyCallComponentWillUnmount(current: Fiber, instance: any) {
```

Comment:
I don't think you need to export this anymore do you since it's called in the same file?